Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ghcr.io/bento-platform/bento_base_image:python-alpine-2022.10.11
+
+# TODO: change USER
+USER root
+
+WORKDIR /notification
+
+# Create data directory
+RUN mkdir -p /notification/data
+
+COPY . /notification
```

Comment:
since `workdir` already changes the cwd, shouldn't this be `COPY . .` ?

Reply:
yes